README.md

```diff
@@ -21,6 +21,8 @@ Start web server
 yarn && yarn build && yarn start
 ```
 
+If you'd like to build with caching with Turbo repo (4x faster, 10x+ faster cached), replace `yarn build` with `yarn build-lint-fast`. To leverage our shared/remote cache, follow [these](https://turbo.build/repo/docs/core-concepts/remote-caching#for-local-development) instructions.
+
```

Comment:
I think that with the recent changes this could be removed

Reply:
Ah meant to update readme. Forgot.